default/python/AI/ResourcesAI.py

```diff
@@ -304,26 +566,26 @@ def print_table_footer(self, priority_ratio):
         all_research_industry_target = 0
         all_research_research_target = 0
         total_changed = 0
-        for pinfo in self.focus_manager.all_planet_info.values():
+        for pinfo in self.focus_manager.baked_planet_info.values():
             if pinfo.current_focus != pinfo.future_focus:
                 total_changed += 1
 
-            old_pp, old_rp, _ = pinfo.possible_output[pinfo.current_focus]
+            old_pp, old_rp, _, _, _ = pinfo.possible_output[pinfo.current_focus]
```

Comment:
There is a short syntax to deal with subsequences.
```python
old_pp, old_rp, *_ = pinfo.possible_output[pinfo.current_focus]`
```

Asterisks could be placed in any part of an expression, but only once per expression.
```python
a, b, *_ = [1, 2, 3, 4, 5]
a, b, _
(1, 2, [3, 4, 5])
```

Reply:
Good to know :)